docs/sources/flow/reference/components/loki.source.gcplog.md

```diff
@@ -80,19 +80,23 @@ a service account key.
 
 ### push block
 
-The `push` block defines the configuration of the HTTP server that receives
+The `push` block defines the configuration of the server that receives
 push requests from GCP's Pub/Sub servers.
 
 The following arguments can be used to configure the `push` block. Any omitted
 fields take their default values.
 
-Name                     | Type          | Description | Default | Required
------------------------- | ------------- | ----------- | ------- | --------
-`http_listen_address`    | `string`      | The address the server listens to.  | `"0.0.0.0"` | no
-`http_listen_port`       | `int`         | The port the server listens to.     |  `8080`     | no
+Name                     | Type          | Description                                                     | Default | Required
+------------------------ |---------------|-----------------------------------------------------------------| ------- | --------
+`http`    | [HTTP][]      | Configures the HTTP server that receives requests.              |  | no
+`grpc`    | [gRPC][]      | Configures the gRPC server that receives requests.              |  | no
```

Comment:
Let's remove these from here as they are not arguments, and document them under the `## Blocks` header.

Their hierarchy would then be `push > http` and `push > grpc`. Also, let's make the `Type` column lowercase, like the block itself is to avoid confusion.

Reply:
Done